sale_stock_info_popup/models/product.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.addons import decimal_precision as dp
+from odoo.exceptions import UserError
+from odoo.tools.float_utils import float_round
+
+import operator as py_operator
+
+OPERATORS = {
+    '<': py_operator.lt,
+    '>': py_operator.gt,
+    '<=': py_operator.le,
+    '>=': py_operator.ge,
+    '=': py_operator.eq,
+    '!=': py_operator.ne
+}
+
+
+class Product(models.Model):
+    _inherit = "product.product"
+
+    """ Based on free_qty field definition in product.product
+        model in Odoo v13 'stock' module. It is necessary for
+        the widget.
+    """
+    free_qty = fields.Float(
```

Comment:
IMHO, as this has been taken away from Odoo, it should use existing code we have in OCA. You should use this : https://github.com/OCA/stock-logistics-warehouse/tree/11.0/stock_available_unreserved

Reply:
No extra modules wants to be used, as this will be merged in core in v13, and having `stock_available_unreserved` in v13 is unwanted.

Reply:
And not only that, the field value for `stock_available_*` family can vary. Any way, we are going to make a bridge with this other module for also showing the potential qty value (whatever it is with the installed modules).

Reply:
Indeed for migration purpose. Ok for me

Reply:
Shouldn't we mention Odoo in licensing ?

Reply:
Yes, I think so. I haven't reviewed the module yet.